tests/test_writer_binary_raw.py

```diff
@@ -152,16 +152,12 @@
     _IT.FLOAT: (
         (None, b'\x4F'),
         (0e0, b'\x40'),
-        (0e-0, b'\x40'),
         (1e0, b'\x48\x3F\xF0\x00\x00\x00\x00\x00\x00'),
-        (1e-0, b'\x48\x3F\xF0\x00\x00\x00\x00\x00\x00'),
         (-1e0, b'\x48\xBF\xF0\x00\x00\x00\x00\x00\x00'),
         (1e1, b'\x48\x40\x24\x00\x00\x00\x00\x00\x00'),
         (-1e1, b'\x48\xC0\x24\x00\x00\x00\x00\x00\x00'),
         (float('inf'), b'\x48\x7F\xF0\x00\x00\x00\x00\x00\x00'),
-        (1e400, b'\x48\x7F\xF0\x00\x00\x00\x00\x00\x00'),
         (float('-inf'), b'\x48\xFF\xF0\x00\x00\x00\x00\x00\x00'),
-        (-1e400, b'\x48\xFF\xF0\x00\x00\x00\x00\x00\x00'),
```

Comment:
These cases and those above it are technically redundant, the Python data model of this text is the same value so we aren't testing a different value.